### ISSUE TITLE
Bug while editing Evaluation Crietria of a challenge

### ISSUE THREAD

**Issue by gautamjajoo** [Cloud-CV/EvalAI#1899]
### Steps to Reproduce
1) Create a challenge
2) Go to ```Evaluation``` tab . Click on edit ```Evaluation Criteria```.
3) Cancel the changes you have made.

### Observed Behaviour
The text displayed is empty.

![screenshot-from-2018-11-10-19-08-45](https://user-images.githubusercontent.com/24366008/48303715-a6d6fc00-e533-11e8-98ad-78c1eea35b96.png)

### Expected Behaviour
The text should be displayed.

![screenshot-from-2018-11-10-19-08-41 1](https://user-images.githubusercontent.com/24366008/48303725-c706bb00-e533-11e8-9663-fac6d9155766.png)

@deshraj @RishabhJain2018

**Comment by supra08** [comment]
@deshraj , @RishabhJain2018 is the issue still open? If yes I would like to work on it.

**Comment by RishabhJain2018** [comment]
Yeah, it is open but I am not sure if this is an issue or not. Can you please reconfirm it.

**Comment by yashdusing** [comment]
@gautamjajoo @RishabhJain2018 I don't think this bug still exists since I was not able to reproduce the bug again

**Comment by Ayukha** [comment]
@RishabhJain2018  @gautamjajoo The bug still exists as I am able to reproduce it.
![out](https://user-images.githubusercontent.com/19167324/52169043-1b4ac100-2758-11e9-94a3-2f8a18a8c571.gif)
